karma.conf.js

```diff
@@ -22,6 +22,12 @@ module.exports = function (config) {
         // web server port
         port: 8888,
 
+        browserConsoleLogOptions: {
```

Comment:
Can you explain what `browserConsoleLogOptions` is used for?

Reply:
It allows to use `console.log` in tests code. I used it to check things in the test I edited to check my fix. I then removed console logs from tests since it isn't a best practice to leave them.